### DISCUSSION TITLE
How do you customize the rootKey used for hashing?

### DISCUSSION THREAD

**Discussion by OwnedTom** [vercel/turborepo#8628] [Help]
### Summary

I'm having an issue where the cache is never hit after building on windows and attempting the same build on Mac, after doing a `--dry=json --summarize` run I noticed that the rootKey is malformed on Windows where it is set to `"rootKey": "I canÔÇÖt see ya, but I know youÔÇÖre here",` instead of `"rootKey": "I can’t see ya, but I know you’re here",`

Is there any way to change this value myself to make it work correctly?

**Comment by anthonyshew** [comment]
We took a look at this and, from what we can tell, that's only a rendering difference. The cache key should be getting interpreted the same in all environments, so something else will be accounting for the difference.

Are you connected to Remote Cache on both machines? Are you seeing the same hash for the runs that you're expecting to hit cache for?
